dissect/target/tools/shell.py

```diff
@@ -512,9 +520,14 @@ def print_extensive_file_stat(self, stdout, target_path: fsutil.TargetPath, name
             entry = entry.fs.get(ads_path)
 
         stat = entry.lstat()
+
+        if entry.is_symlink():
+            symlink = f"-> {entry.readlink()}"
```

Comment:
```suggestion
        symlink = f"-> {entry.readlink()}" if entry.is_symlink() else ""
```
You use a one liner elsewhere, might as well also use it here.